x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint/automated_response_actions.cy.ts

```diff
@@ -74,35 +74,16 @@ describe('Automated Response Actions', () => {
 
       changeAlertsFilter('event.category: "file"');
       cy.getByTestSubj('expand-event').first().click();
-      cy.getByTestSubj('endpointViewTab').click();
-      cy.getByTestSubj('endpoint-actions-notification').should('not.have.text', '0');
-      cy.getByTestSubj('endpoint-actions-results-table').within(() => {
-        cy.get('tbody tr').each(($tr, index) => {
-          cy.wrap($tr).within(() => {
-            cy.contains('Triggered by rule');
-            if (index === 0) {
-              cy.getByTestSubj('endpoint-actions-results-table-column-hostname').contains(
-                endpointHostname
-              );
-              cy.getByTestSubj('endpoint-actions-results-table-column-status', {
-                timeout: 60000,
-              }).should(($status) => {
-                expect($status).to.contain('Successful');
-              });
-              cy.getByTestSubj('endpoint-actions-results-table-expand-button').click();
-              cy.wrap($tr).siblings().contains('isolate completed successfully');
-            }
-            if (index === 1) {
-              cy.getByTestSubj('endpoint-actions-results-table-column-hostname').contains(
-                unenrolledHost
-              );
-              cy.getByTestSubj('endpoint-actions-results-table-column-status').contains('Failed');
-              cy.getByTestSubj('endpoint-actions-results-table-expand-button').click();
-              cy.wrap($tr)
-                .siblings()
-                .contains('The host does not have Elastic Defend integration installed');
-            }
-          });
+      cy.getByTestSubj('responseActionsViewTab').click();
+      cy.getByTestSubj('response-actions-notification').should('not.have.text', '0');
+      cy.getByTestSubj('responseActonsViewWrapper').within(($wrapper) => {
+        cy.wrap($wrapper).each(($comment, index) => {
+          if (index === 0) {
+            cy.contains('isolate completed successfully');
```

Comment:
can we find a more reliable way to check the proper output for the action type than relying on the order?

Reply:
I have no other ideas how to tackle this. I can just check that one was ok, the second bounced. If we decide to add agent details, then it might be easier. What do you think?
![Zrzut ekranu 2023-05-16 o 12 44 08](https://github.com/elastic/kibana/assets/16632552/0ff80629-52ab-4482-bfbd-351002c10c50)